ingestion/src/metadata/ingestion/source/pipeline/airflow/connection.py

```diff
@@ -114,8 +128,39 @@ def test_connection(
     of a metadata workflow or during an Automation Workflow
     """
 
-    test_fn = {"CheckAccess": partial(test_connection_engine_step, engine)}
-
+    session_maker = sessionmaker(bind=engine)
+    session = session_maker()
+
+    def test_pipeline_details_access(session):
+        try:
+            result = session.query(SerializedDagModel).limit(1)
```

Comment:
instead of using limit, we could use first

Reply:
Did the required changes, tested using CLI and UI, works fine.